src/glyph/mod.rs

```diff
@@ -29,7 +29,8 @@ use crate::{Color, Guideline, Identifier, Line, Plist, WriteOptions};
 #[cfg_attr(feature = "druid", derive(Lens))]
 pub struct Glyph {
     /// The name of the glyph.
-    pub name: Name,
+    #[cfg_attr(feature = "druid", lens(ignore))]
```

Comment:
```suggestion
```

I don't *think* this is necessary, I think non-pub items are ignored by default? In any case I can fix this up if/when I pick up runebender next.

Reply:
I get a clippy/test failure unless I put that line in, because the macro seems to want to make a name() function, which already exists.